Size frequency modifications to the manual detailing Dirichlet and MV Tweedie data fitting

chantelwetzel-noaa opened the pull request:
I followed the example data file and the information in data file reading in the [MV-Tweedie branch SS_write_ssnew file](https://github.com/nmfs-stock-synthesis/stock-synthesis/blob/MV-Tweedie-ver2/SS_write_ssnew.tpl). The changes to the manual are rather minor but I am not sure if there are additional edits to describe this new option within SS3.

iantaylor-NOAA reviewed (approved):
Thanks for working on this @chantelwetzel-noaa. The changes look good overall, but there are some formatting things that make this hard to read, at least in the HTML view.

First, can we make the first few columns wider so numbers don't wrap as shown in the snippet below?

Second, the conditional stuff is confusing in the User Manual in general, but especially hard here. What about changing from "N of size frequency COND < 0" to making those entries in italics (or colored text, but maybe that's too fancy) and adding the italic text to the first entry: "A value of -1 (or any negative value) triggers expanded optional inputs below that allow for either Dirichlet of two parameter Multivariate (MV) Tweedie likelihood for fitting these data. _These expanded inputs are shown in italics_."

![image](https://user-images.githubusercontent.com/4992918/192629998-2d63a524-fe5e-48c2-97bb-323931ad9adb.png)

chantelwetzel-noaa commented:
I have done the following edits:

1. Remove the conditional statement for the needed entries to include generalize size frequency data (COND > 0) and instead rely on the text in the initial specification line which states to skip all additional lines if set to 0.
2. Widen the second column to reduce wrapping in the html. The larger issue here is that these values were NOT wrapping in the pdf which is how I iteratively visualize manual edits where, in contrast, the html version is only viewable post pushed commits via gh-actions.
3. Added lines to create visual separation for the COND entries (unfortunately horizontal lines are not being rendered anywhere within the html version) and added a specific END COND lines for additional clarity.
